backend/dataall/base/utils/naming_convention.py

```diff
@@ -4,10 +4,22 @@
 
 
 class NamingConventionPattern(Enum):
-    S3 = {'regex': '[^a-zA-Z0-9-]', 'separator': '-', 'max_length': 63}
+    S3 = {
+        'regex': '[^a-zA-Z0-9-]',
+        'separator': '-',
+        'max_length': 63,
+        'valid_external_regex': '^[a-z0-9][a-z0-9.-]{1,61}[' 'a-z0-9]$',
+        'max_imported_length': 63,
+    }
+    KMS = {'regex': '[a-zA-Z0-9/_-]+$', 'separator': '-', 'max_length': 63}
```

Comment:
do we need a `/` as a supported character? If so do we need to escape it with `\/`?

Reply:
We don't need it, true